pkgpanda/docker.windows/dcos-builder/setup-python36.ps1

```diff
@@ -0,0 +1,30 @@
+$pip_url = 'https://bootstrap.pypa.io/get-pip.py'
```

Comment:
Should this script start with `$ErrorActionPreference = "stop"`?

Reply:
It looks like there won't be any valuable output if we add error handling there. It won't stop the build of the Docker container. It would be done anyway but without Python within.